src/Sarif/LineInfo.cs

```diff
@@ -86,6 +86,7 @@ public override string ToString()
         /// <param name="lhs">The left hand side.</param>
         /// <param name="rhs">The right hand side.</param>
         /// <returns>The result of the operation.</returns>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "rhs")]
```

Comment:
There's actually a .NET Framework Design Guidelines section on this one:

> **DO** use _left_ and _right_ for binary operator overload parameter names if there is no meaning to the parameters.

So rename `lhs` to `left`, `rhs` to `right`, and remove the suppression.